sdk/cosmosdb/cosmos/src/globalEndpointManager.ts

```diff
@@ -175,7 +175,7 @@ export class GlobalEndpointManager {
         this.writeableLocations.push(location);
       }
     }
-    for (const location of databaseAccount.writableLocations) {
+    for (const location of databaseAccount.readableLocations) {
```

Comment:
@southpolesteve do you think this was a bug? this causes test failures

Reply:
I am not sure. Did this method work before this change? Let's discuss tomorrow